src/main/java/com/vmware/vipclient/i18n/base/HttpRequester.java

```diff
@@ -50,17 +51,23 @@
      */
     private String                  baseURL;
 
-    public void setBaseURL(String baseURL) {
-        this.baseURL = baseURL;
-    }
-
     /**
-     * The extra parameters to add to http header
+     * HTTP headers that are common to all HTTP requests
      */
-    private Map<String, String> customizedHeaderParams = null;
+    private Map<String, String> commonHeaderParams = null;
 
     public void setCustomizedHeaderParams(Map<String, String> params) {
-        customizedHeaderParams = params;
+    	if (params!=null && !params.isEmpty()) {
+    		if (commonHeaderParams == null) {
+    			commonHeaderParams = new HashMap<String, String>();
```

Comment:
Why not assign params to commonHeaderParams directly?

Reply:
Okay, I will bring this back to the original implementation and assign directly.